ui/app/AppLayouts/Profile/views/NotificationsView.qml

```diff
@@ -1,399 +1,576 @@
-                }
+            StatusBaseText {
+                Layout.preferredWidth: root.profileContentWidth
+                Layout.leftMargin: Style.current.padding
+                text: qsTr("Notifications & Sounds")
+                font.weight: Font.Bold
+                font.pixelSize: Constants.settingsSection.mainHeaderFontSize
+                color: Theme.palette.directColor1
             }
 
-            // TODO: replace with StatusListItem
-            StatusSettingsLineButton {
-                //% "Use your operating system's notifications"
-                text: qsTrId("use-your-operating-system-s-notifications")
-                isSwitch: true
-                switchChecked: localAccountSensitiveSettings.useOSNotifications
-                onClicked: {
-                    localAccountSensitiveSettings.useOSNotifications = checked
-                }
+            Rectangle {
+                Layout.preferredWidth: root.profileContentWidth
+                implicitHeight: col1.height + 2 * Style.current.padding
+                visible: Qt.platform.os == "osx"
+                radius: Constants.settingsSection.radius
+                color: Theme.palette.primaryColor3
 
-                StatusBaseText {
-                    id: detailText
-                    //% "Setting this to false will instead use Status' notification style as seen below"
-                    text: qsTrId("setting-this-to-false-will-instead-use-status--notification-style-as-seen-below")
-                    color: Theme.palette.baseColor1
-                    width: parent.width
-                    font.pixelSize: 12
-                    wrapMode: Text.WordWrap
+                ColumnLayout {
+                    id: col1
+                    anchors.margins: Style.current.padding
                     anchors.left: parent.left
-                    anchors.leftMargin: Style.current.padding
-                    anchors.bottom: parent.bottom
-                    anchors.topMargin: 2
+                    anchors.right: parent.right
+                    anchors.verticalCenter: parent.verticalCenter
+                    spacing: Constants.settingsSection.infoSpacing
+
+                    StatusBaseText {
+                        Layout.preferredWidth: parent.width
+                        text: qsTr("Enable Notifications in macOS Settings")
+                        font.pixelSize: Constants.settingsSection.infoFontSize
+                        lineHeight: Constants.settingsSection.infoLineHeight
+                        lineHeightMode: Text.FixedHeight
+                        color: Theme.palette.primaryColor1
+
+                    }
+
+                    StatusBaseText {
+                        Layout.preferredWidth: parent.width
+                        text: qsTr("To receive Status notifications, make sure you've enabled them in" +
+                                   " your computer's settings under <b>System Preferences > Notifications</b>")
```

Comment:
Since we might disable RichText by default, it might be safer to add the RichText mode here in case.

Reply:
@jrainville in this particular case we don't have `RichText` explicitly set, it's left to the `StatusBaseText` component to decide what to do, but if you see beneath, it's just a simple `Text` component so `Text.AutoText` will be used as that's default for `Text`, which is I guess fine. Otherwise if we ever decide to set explicitly `textFormat: Text.PlainText` in `StatusBaseText` that will be a breaking change which will affect lot of places (hope that won't happen ever :) ).

Reply:
> Otherwise if we ever decide to set explicitly textFormat: Text.PlainText in StatusBaseText that will be a breaking change which will affect lot of places (hope that won't happen ever :) ).

Yeah that was my point. We've had a couple of "XSS" issues that we caused by Texts accepting HTML. We proposed the change to never allow HTML soit doesn't happen again.
Like you pointed out, it would be a breaking change, so maybe we don't do it. 
I was just proposing that you set it manually here, just in case we ever do change it.